scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -171,7 +171,8 @@ export function AuthoringDirective(
              * @returns {Boolean}
              */
             $scope.canPublishOnDesk = function() {
-                return !($scope.deskType === 'authoring' && config.features.noPublishOnAuthoringDesk);
+                return !($scope.deskType === 'authoring' && config.features.noPublishOnAuthoringDesk) &&
+                    !!$scope.privileges.publish;
```

Comment:
Is `!!$scope.privileges.publish` meant to mean `$scope.privileges.publish != null`? Use the explicit version.